posthog/api/event.py

```diff
@@ -78,6 +81,18 @@ def _build_next_url(self, request: request.Request, last_event_timestamp: dateti
                 OpenApiTypes.STR,
                 description="Filter list by event. For example `user sign up` or `$pageview`.",
             ),
+            OpenApiParameter(
+                "select",
+                OpenApiTypes.STR,
+                description="JSON-serialized array of HogQL expressions to return",
```

Comment:
Let's add a note that this is `(Experimental)` - indicating that this may change soon?